include/eve/algo.hpp

```diff
@@ -0,0 +1,55 @@
+//==================================================================================================
```

Comment:
Why would we need this file?
It's not how algo is suppose to be used.

Reply:
This is the transitional module like include.